Cargo.toml.jinja

```diff
@@ -3,7 +3,7 @@ name = "frequenz-{{project_name}}"
 description = "{{project_description}}"
 version = "0.1.0"
 edition = "2021"
-authors = ["Frequenz Energy-as-a-Service GmbH"]
+authors = ["{{project_author}}"]
```

Comment:
Nitpick: This looks unrelated to adding the python CI so it could be split in a different commit.

Reply:
This raises another question - who do we say the author is? The individual(s), or the team, or the company? Do you already use a standard on the python side of things?